READMEs/setup-local.md

```diff
@@ -25,6 +25,9 @@ cd barge
 # Clean up old containers (to be sure)
 docker system prune -a --volumes
 
+# Give permissions to the script
+sudo chmod +x start_ocean.sh
```

Comment:
please wrap this in a console or at least `code` markdown. I think the instruction above should be the same

Reply:
no, sorry. I missed the generic wrap :)